src/test_application/OSPMagnum.cpp

```diff
@@ -138,11 +139,11 @@ void OSPMagnum::mouseScrollEvent(MouseScrollEvent & event)
 
 osp::active::ActiveScene& OSPMagnum::scene_add(const std::string &name)
 {
-    auto pair = m_scenes.try_emplace(name, m_userInput, m_ospApp);
-    return pair.first->second;
+    auto const& [it, success] = m_scenes.try_emplace(name, m_userInput, m_ospApp);
```

Comment:
Consider leaving a comment saying that if the scene is already there, don't want to add a new one, just return the existing one.

Which, that being said, maybe the name of the function should be "acquire_scene" or something similar, so that it doesn't sound like it's allocating a new scene.